CorsixTH/Lua/humanoid_actions/seek_reception.lua

```diff
@@ -20,15 +20,18 @@ SOFTWARE. --]]
 
 local function can_join_queue_at(humanoid, x, y, use_x, use_y)
   local flag_cache = humanoid.world.map.th:getCellFlags(x, y)
-  return flag_cache.hospital and not flag_cache.room
+  return flag_cache.hospital and not flag_cache.room and
+    humanoid.hospital and
```

Comment:
Continuation lines have double indent. I'd also merge line 25 after this one, but that's optional :)